paywall/package.json

```diff
@@ -9,7 +9,7 @@
     "build": "npm run before && next build src",
     "deploy": "next export src -o out",
     "deploy-netlify": "./scripts/deploy-netlify.sh",
-    "start": "npm run before && next start",
+    "start": "cross-env NODE_ENV=production node src/server.js",
```

Comment:
I strongly think we should not set env variables in these commands... any way we can avoid that?

Reply:
perhaps. But for consistency, I think we should use this. It's a literal copy/paste from `unlock-app`. Certainly, in a follow-up PR, we could address it in both apps, but that is outside the narrow scope of this particular PR. What would you like me to do?

Reply:
Ok, please open an issue about this then...